src/main/java/model/BlindHorizontal.java

```diff
@@ -1,20 +1,20 @@
 package model;
 
-public class BlindHorizontal {
+public class BlindHorizontal extends Blind{
 
     private int width;
```

Comment:
need to remove fields exist in superclass

Reply:
done